_data/retail.yml

```
@@ -416,6 +416,16 @@ websites:
     facebook: staples
     img: staples.png
 
+  - name: STEG Electronics
+    url: https://www.steg-electronics.ch
+    img: steg.png
+    twitter: stegelectronics
```

Like the error message said:
```suggestion
    twitter: STEGelectronics
```

---

Thank you, I have no access to the [workflow tests](https://circleci.com/gh/2factorauth/twofactorauth/1770?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) so I can't see the error message.

---

It doesn't allow you to see the logs even when you've logged in? Odd 🤔